src/config/ch.sbb.netzkarte.sandbox/index.js

```diff
@@ -26,21 +25,21 @@ export const poiLayer = new MapboxStyleLayer({
   style: "temp_entwicklungsstyle",
 });
 
-export const geschosseLayer = new Layer({
+export const sandboxGeschosseLayer = new Layer({
   name: "ch.sbb.geschosse",
   visible: true,
+  properties: {
+    hideInLayerTree: true,
```

Comment:
Use `hideInLegend` instead of adding a new property

Reply:
because hideInLegend also removes the layers from the permalink, so we can't control the levels via search params